### PR TITLE
Add recur, unrecur subcommands

### PR DESCRIPTION
**Pull request by dlecocq**

### CODE REVIEW

File: bin/qless-js.js [hunk 1]
```diff
@@ -247,6 +281,11 @@ commander
   .description('Cancel all the provided jobs')
   .action(cancel);
 
+commander
+  .command('unrecur <jid>')
+  .description('Unrecur all the provided job')
```
Comment (evanbattaglia) [inline bin/qless-js.js:286]: s/job/jobs/

File: bin/qless-js.js [hunk 2]
```diff
@@ -210,11 +231,24 @@ commander
...
-  .option('-t, --tags <tag>', 'Add a tag to the job', collect)
+  .option('-t, --tags <tag>', 'Add a tag to the job', collect, [])
   .option('-r, --retries <retries>', 'The number of retries to use for the job', parseInt)
-  .option('-s, --depends <depends>', 'Add a job on which it depends', collect)
+  .option('-s, --depends <depends>', 'Add a job on which it depends', collect, [])
   .action(put);
 
+commander
+  .command('recur <name> <next>')
+  .description('Put a job in the provided queue')
```
Comment (evanbattaglia) [inline bin/qless-js.js:241]: Description seems to be inaccurate (it's the same as `put`)

File: bin/qless-js.js [hunk 3]
```diff
@@ -60,6 +60,20 @@ const put = (name, options) => {
   logWithClient(options.parent, client => client.queue(name).put(options));
 };
 
+/*
+ * Recur a job
+ */
+const recur = (name, next, options) => {
+  logWithClient((client) => {
+    const timeNow = now();
```
Comment (evanbattaglia) [inline bin/qless-js.js:68]: I don't see `now` defined anywhere? I assume you tested it and it works so I assume I'm just missing it

File: bin/qless-js.js (hunk 1 above)
Comment (dlecocq) [reply bin/qless-js.js:286]: Whoops

File: bin/qless-js.js (hunk 2 above)
Comment (dlecocq) [reply bin/qless-js.js:241]: Copypasta

File: bin/qless-js.js (hunk 3 above)
Comment (dlecocq) [reply bin/qless-js.js:68]: Copypasta

**Comment by evanbattaglia** [comment]
I'm really not used to github anymore... it wouldn't even let me see the old commit I commented on in full (because you amended/force pushed I guess), even though it shows a snippet of it...

**Pull request merged**